.github/workflows/ci.yml

```diff
@@ -7,38 +7,33 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+        with:
+          enable-cache: true
+
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
```

Comment:
Can we let uv manage Python too?

Reply:
I'm inclined to use `setup-python`, as is recommended by https://docs.astral.sh/uv/guides/integration/github/#setting-up-python, since GitHub manages caching

Reply:
If you feel strongly about this we can change it afterwards, but let's see if publishing works